src/components/TopBar.jsx

```diff
@@ -48,6 +49,12 @@ const useStyles = makeStyles(theme => ({
 		width: '20px',
 		height: '20px',
 		padding: '0 4px 4px 0'
+	},
+	alert: {
+		width: '50%',
+		'& > * + *': {
```

Comment:
Oha, what is this?

Reply:
Ah, that was me experimenting some stuff. I moved the warning to a different component. Regarding the text centering - i did a temporary fix  >.< That will break if you open the dev tools and change the width of the page. 
The previous one was working fine even with varying width of the page. But for some reason the text is shifted to a bit left in normal view.